src/components/Settings/SettingsPlex.tsx

```diff
@@ -108,14 +110,26 @@ const SettingsPlex: React.FC<SettingsPlexProps> = ({ onComplete }) => {
   const { addToast, removeToast } = useToasts();
   const PlexSettingsSchema = Yup.object().shape({
     hostname: Yup.string()
+      .nullable()
       .required(intl.formatMessage(messages.validationHostnameRequired))
       .matches(
         /^(([a-z]|\d|_|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*)?([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])$/i,
         intl.formatMessage(messages.validationHostnameRequired)
       ),
     port: Yup.number()
-      .typeError(intl.formatMessage(messages.validationPortRequired))
+      .nullable()
```

Comment:
Why is port nullable? 🤔

Reply:
It's not that it's nullable, it's just so that null values are not considered type errors. It's still a required field.  See https://github.com/jquense/yup#mixedrequiredmessage-string--function-schema